### ISSUE TITLE
Improve Windows install instructions

### ISSUE THREAD

**Issue by EnforcerRidley** [tralph3/Steam-Metadata-Editor#30]
How do I run this on Windows? The info in the Readme are not clear.

**Comment by Tojaw** [comment]
You need to open a terminal window and type `python main.py`.
Alternatively, you can create a shortcut and set this as the target.

**Comment by EnforcerRidley** [comment]
That worked! Thanks!

**Comment by tralph3** [comment]
There's not much I can do with "it doesn't work". Try launching it via terminal and see if there's any output.

**Comment by TheWaslijn** [comment]
> You need to open a terminal window and type python main.py.

What does that mean?

**Comment by GLaDOS49** [comment]
> What does that mean? Also, I cannot seem to get a shortcut working.

I think it means cmd prompt, but I'm not sure.

**Comment by Tojaw** [comment]
1. Locate the `main.py` file in the folder you extracted
2. Right-click and click "Copy as path"
3. Press `Win + X` or right-click the Start icon
4. Click either "Terminal" or "PowerShell" or Command line" or something like that
5. Type `python ` followed by the path you copied and press enter

To create a shortcut, right-click the Desktop, select "New" and "Shortcut". Enter `python ` followed by the copied path. Click "Continue", type a name of your choice and finish the Shortcut wizard.

**Comment by nondetect** [comment]
> 5. Type `python ` followed by the path you copied and press enter
> 
> To create a shortcut, right-click the Desktop, select "New" and "Shortcut". Enter `python ` followed by the copied path. Click "Continue", type a name of your choice and finish the Shortcut wizard.

should be mentioned, that Python needed first to be installed in system

**Comment by Tojaw** [comment]
> should be mentioned, that Python needed first to be installed in system

True. On the other hand, the installation instructions are pretty clear about that. Still, gonna update the post, maybe people have already forgotten that.
